### ISSUE TITLE
Compilation fails for `master` branch, also using `cargo install`

### ISSUE THREAD

**Issue by mbr** [kennytm/cov#22]
```
   Compiling tera v0.11.9
error[E0308]: mismatched types
  --> cargo-cov/src/error.rs:11:1
   |
11 | / error_chain! {
12 | |     links {
13 | |         Cov(::cov::error::Error, ::cov::error::ErrorKind);
14 | |         Tera(::tera::Error, ::tera::ErrorKind);
...  |
47 | |     }
48 | | }
   | |_^ expected struct `error_chain::State`, found a different struct `error_chain::State`
   |
   = note: expected type `error_chain::State` (struct `error_chain::State`)
              found type `error_chain::State` (struct `error_chain::State`)
note: Perhaps two different versions of crate `error_chain` are being used?
  --> cargo-cov/src/error.rs:11:1
   |
11 | / error_chain! {
12 | |     links {
13 | |         Cov(::cov::error::Error, ::cov::error::ErrorKind);
14 | |         Tera(::tera::Error, ::tera::ErrorKind);
...  |
47 | |     }
48 | | }
   | |_^
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error[E0308]: mismatched types
  --> cargo-cov/src/error.rs:11:1
   |
11 | / error_chain! {
12 | |     links {
13 | |         Cov(::cov::error::Error, ::cov::error::ErrorKind);
14 | |         Tera(::tera::Error, ::tera::ErrorKind);
...  |
47 | |     }
48 | | }
   | |_^ expected enum `std::option::Option`, found struct `error_chain::backtrace::imp::InternalBacktrace`
   |
   = note: expected type `std::option::Option<std::sync::Arc<error_chain::Backtrace>>`
              found type `error_chain::backtrace::imp::InternalBacktrace`
   = note: this error originates in a macro outside of the current crate (in Nightly builds, run with -Z external-macro-backtrace for more info)

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0308`.
error: Could not compile `cargo-cov`.

To learn more, run the command again with --verbose.
```

I would assume a version mismatch/API change (currently both `error-chain` 0.11 and 0.12 are used in the build). Maybe checking in `Cargo.lock` could solve this issue?

**Comment by garious** [comment]
@kennytm, the fix in #23 worked for me. Can you push a release to crates.io?

**Comment by kennytm** [comment]
Sorry. Published 0.0.5 now.
